inc/editor/blocks/blog-post.php

```diff
@@ -50,7 +50,11 @@ function render_block( $attributes ) {
 	}
 
 	$post_query = new \WP_Query( [
-		'post_type' => 'post',
+		'post_type' => [ 'post', 'page' ],
+		'orderby'   => [
+			'relevance' => 'DESC',
```

Comment:
Does `orderby=relevance` do anything when not performing a search query?

Actually, I don't think any of these parameters are going to be used, since there's a single post ID specified below. Why is this forcing a database query and not just doing a `get_post( $id )`?

Reply:
Your comment made me implement it differently, thanks for the heads up.